master/manifests/site.pp

```diff
@@ -244,59 +169,58 @@
     notify => Service['jenkins'],
 }
 
-jenkins::plugin {
-  'purge-build-queue-plugin': ;
-}
-
+jenkins::plugin { 'purge-build-queue-plugin': ; }
 # required by mercurial
-jenkins::plugin {
-  'scm-api': ;
-}
-
+jenkins::plugin { 'scm-api': ; }
 # required by groovy-postbuild
-jenkins::plugin {
-  'script-security': ;
-}
-
-jenkins::plugin {
-  'ssh-agent': ;
-}
-
-jenkins::plugin {
-  'subversion': ;
-}
-
-jenkins::plugin {
-  'systemloadaverage-monitor': ;
-}
-
-jenkins::plugin {
-  'swarm':
+jenkins::plugin { 'script-security': ; }
+# required by groovy-postbuild
+# required by groovy-postbuild
+jenkins::plugin { 'durable-task': ; }
+# required by groovy-postbuild
+jenkins::plugin { 'ssh-agent': ; }
```

Comment:
We need `ssh-agent` for sure but where did you find that it is a requirement for `groovy-postbuild`?